### PR TITLE
[OPS-7730] fix nginx error

### PR DESCRIPTION
**Pull request by teodorescuserban**
Tested and applied. This essentially moves the definition of $connection_upgrade _after_ inclusion of /etc/http.d conf files.

### CODE REVIEW

**Comment by teodorescuserban** [comment]
We might find that adding the map definition after the http.d is basically useless as we cannot use it in the our custom config and will need to re-define it (under a different name) but meh.

File: nginx/Dockerfile.tmpl [hunk 1]
```diff
@@ -36,10 +36,20 @@ RUN apk update && \
...
+    echo "include /etc/nginx/env/*.conf;" >> /etc/nginx/nginx.conf && \
+    sed -i "s|^\tmap \$http_upgrade|\t# map \$http_upgrade|" /etc/nginx/nginx.conf && \
+    sed -i "s|^\t\tdefault upgrade;|\t#\tdefault upgrade;|" /etc/nginx/nginx.conf && \
+    sed -i "s|^\t\t'' close;|\t#\t'' close;|" /etc/nginx/nginx.conf && \
+    sed -i "s|^\t}|\t# }|" /etc/nginx/nginx.conf && \
+    sed -i "s|http.d/\*\.conf;|http.d/\*\.conf;\n\t}|" /etc/nginx/nginx.conf && \
+    sed -i "s|http.d/\*\.conf;|http.d/\*\.conf;\n\t\t'' close;|" /etc/nginx/nginx.conf && \
+    sed -i "s|http.d/\*\.conf;|http.d/\*\.conf;\n\t\tdefault upgrade;|" /etc/nginx/nginx.conf && \
+    sed -i "s|http.d/\*\.conf;|http.d/\*\.conf;\n\tmap \$http_upgrade \$connection_upgrade {|" /etc/nginx/nginx.conf && \
+    sed -i "s|http.d/\*\.conf;|http.d/\*\.conf;\n\n\t# Helper variable for proxying websockets.|" /etc/nginx/nginx.conf
```
Comment (cafuego) [inline nginx/Dockerfile.tmpl]: I don't understand these. We can just leave this section out, can't we?

Comment (teodorescuserban) [reply nginx/Dockerfile.tmpl]: Yes I think so, it is pretty much useless inside the http.d as it is defined after.
I'll remove it.

Comment (teodorescuserban) [reply nginx/Dockerfile.tmpl]: Done. Also built a new image and aliased it to 1.20 so it can be used immediately.

**Review by cafuego — APPROVED** [review]

**Pull request merged**